Add GUI/application intro (#36)
napari/napari-workshop-template: napari-workshops/intro_napari_GUI.md

* Add a points, shapes, or labels layer using the buttons above the layers list.
    * These layers will be blank to begin with.  
    * In 2D mode, use the layer controls to add points, draw some shapes, or paint some labels on your image (referring to points, shapes, and labels layers respectively).
* Re-order the layer list by dragging a selected layer in the list below another. Depending on the blending and opacity settings, this will change the visualization!

```{tip}
You can delete layers selected individually using the trash can icon or by pressing the backspace key.
Delete all layers in the layer list by selecting one layer, and then pressing Command+A(Mac) / Control+A(Windows) to select all layers. Then click the trash can button on the top right of the layer list. 
```

## Save your data

There are two options for saving your data: saving the layers themselves or saving the rendered view
in the canvas. 

To save the currently rendered view, use the File menu **Save Screenshot**. You can also chose to include
the viewer or to save the screenshot to the clipboard instead of a file.

> You can save the layers individually using the File menu or Command+S(Mac) / Control+S(Windows). 

Reviewer: ```suggestion
You can save the layers individually using the File menu or Command+S (Mac) / Control+S (Windows and Linux). 

```

While checking this I also opened https://github.com/napari/napari/issues/7387

* For image layers, you can select the file type by providing the extension. 
Note: Some formats may not work for all image types, e.g. 3D stacks. Additional file types 
can be made available by plugins.
* Points and Shapes layers are saved as `.csv` files that contain their coordinates.
* Labels layers are saved as `.tiff` files.

Multiple different layer types, e.g. image and points, can be saved to a single `.svg` file,
which can be convenient for use in a vector graphics program or for web. 

> * Select more than one layer and use the File menu or Command+S(Mac) / Control+S(Windows), 

Reviewer: ```suggestion
* Select more than one layer and use the File menu or Command+S (Mac) / Control+S (Windows and Linux), 

```

> then in the dialog dropdown menu ensure `napari-svg` is selected, give the file a name, and save.

Reviewer: ```suggestion
then in the dialog dropdown menu ensure `napari SVG` is selected, give the file a name, and save.

```

    ```{important} 
    The output file will not be openable in napari!
    ```

Perhaps more usefully, multiple different layer types, e.g. image and points, can be saved as
individual layers (as described above) *to a folder.* 

> * Select more than one layer and use the File menu or Command+S(Mac) / Control+S(Windows) or use Command+Shift+S(Mac) / Control+Shift+S(Windows) to save all layers. Then in the dialog dropdown menu change `napari-svg`

Reviewer: ```suggestion
* Select more than one layer and use the File menu or Command+S (Mac) / Control+S (Windows and Linux) or use Command+Option+S (Mac) / Control+Alt+S (Windows and Linux) to save all layers. Then in the dialog dropdown menu change `napari-svg`

```

The Control+Shift+S shortcut did not work for me in Linux, neither did the Command+Shift+S on a Mac, so I tested and it's actually an Option/Alt shortcut (the menu lists the correct shortcut in both systems)

![console-button](resources/console-button.png)

    ```{important}
    The integrated Python console is only available if napari was started from a non-interactive session, meaning the terminal (using `napari`) or a script. If you are already in an interactive session, like iPython or a notebook, keep using that!
    ```
* Get the layer list programmatically; enter:

    ```python
    viewer.layers
    ```

* Adjust the scale of an image. If you have just the `nuclei` and `membrane` image layers, you can provide micron to pixel scale information for the `nuclei` layer as follows:

    ```Python
    viewer.layers['nuclei'].scale = [0.35, 0.2, 0.2]
    ```

Alternatively, you could use the index of the layer instead. 

* If you want to change the scale of all layers, enter the following commands one line at a time, ensuring an indent on the second line, as shown below: 

    ```python
    for layer in viewer.layers:  
        layer.scale = [0.35, 0.2, 0.2]
    ```

```{important} 
If your layers disappeared, you will need to click the `home` button to reset the viewer and you may need to adjust the slider to a new slice
```

> * Add a scale bar using the GUI **View** > **Scale Bar** > **Visible** or by typing 

Reviewer: ```suggestion
* Add a scale bar using the GUI **View** > **Scale Bar** > **Scale Bar Visible** or by typing 

```

`viewer.scale_bar.visible = True`  in the integrated console. 
* Add physical units of microns to the scale bar by typing `viewer.scale_bar.unit = "um"`  
in the integrated console.
* Try zooming in and out of the image, while watching the scale bar!

````{tip}
The integrated console is a full-featured iPython kernel. You can use it for interactive
analysis, just import `numpy` or `skimage`. You can use `tab` for auto-completions and the Up

> arrow to access the command history. Use a ? to access documentation, e.g.

Reviewer: ```suggestion
arrow to access the command history. Use a `?` to access documentation, e.g.

```
